eng/targets/Settings.props

```diff
@@ -43,6 +43,9 @@
     <WindowsAppContainer>false</WindowsAppContainer>
 
     <EnableDefaultNoneItems>false</EnableDefaultNoneItems>
+ 
+    <!-- TODO: Remove suppression. https://github.com/dotnet/roslyn/issues/38471 -->
```

Comment:
Didn't we already have this disabled in a ruleset file somewhere? @sharwell?

Reply:
I see. We have VSSDK001 and VSSDK003 suppressed in ruleset file. I though we consider ruleset files obsolete.

Reply:
There are multiple locations to suppress, but we should keep them together (i.e. use ruleset for now since that's what already gets used)